### ISSUE TITLE
Git - Show full commit message when hovering over blame

### ISSUE THREAD

**Issue by mjbvz** [microsoft/vscode#235191]
Testing #235028

I think it would be useful to see the entire commit message when hovering over blame annotation. Most of the subject is already shown in the editor so showing it again in the hover feels redundant

**Comment by lszomoru** [comment]
Agreed! The git blame output only contains the git commit subject so displaying the full commit messages requires more work to be done. I will investigate how to add the full commit message to the editor decoration hover and the status bar item hover.

**Comment by lszomoru** [comment]
This has been implemented for the editor decoration but not for the status bar item (blocked on https://github.com/microsoft/vscode/issues/234339).

**Comment by lszomoru** [comment]
Verification steps:
* Launch VS Code Insiders
* Open a folder/workspace that contains a git repository
* Open settings and toggle the `git.blame.editorDecoration.enabled` and `git.blame.statusBarItem.enabled` settings
* Open a file, make changes to it, and commit the changes with a commit message that contains multiple lines
* Place the cursor on the line of the change that was committed
* Hover over the editor decoration
    * Confirm that the hover contains the full commit message with all lines
* Hover over the status bar item
    * Confirm that the hover contains the full commit message with all lines
